develop-docs/frontend/using-rtl.mdx

```diff
@@ -97,6 +99,91 @@ const {router} = render(<TestComponent />, {
 expect(screen.getByText('123')).toBeInTheDocument();
 ```
 
+### Outlet Context
+
+If you need to test outlet param values (as in `useOutletContext()`), you can pass them inside the `initialRouterConfig` object.
+
+```tsx
+function TestComponent() {
+  const { id } = useOutletContext();
```

Comment:
We should suggest they use a typed hook no? These return `unknown`.

Really they should be typed as

```tsx
interface OutletContext {
  id: string
}

export function useWhateverOutletContext() {
  return useOutletContext<OutletContext>();
}

export function MyParentView() {
  return <Outlet id="whatever" />
}
```

Reply:
ah yeah. i can add that in so it's most copy+pasteable